chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/schedule/ChplTrigger.java

```diff
@@ -17,6 +17,7 @@
     private ChplJob job;
     private String cronSchedule;
     private String email;
+    private String acb;
```

Comment:
I believe we allow ACBs to change their names and since this value is a string I'm assuming it's the name rather than the ID. Should we be using the ID here instead to cover ourselves when one ACB changes its name?